src/Image.tsx

```diff
@@ -118,23 +121,27 @@ const ImageInternal: React.FC<ImageProps> = ({
     srcSet,
     useMap,
     alt,
-    className: cn(`${prefixCls}-img`, {
-      [`${prefixCls}-img-placeholder`]: placeholder === true,
-    }),
-    style: height !== undefined ? { height } : undefined,
+    className: cn(
+      `${prefixCls}-img`,
+      {
+        [`${prefixCls}-img-placeholder`]: placeholder === true,
+      },
+      className,
+    ),
+    style: {
+      height,
+      width,
+      ...style
+    },
   };
 
   return (
     <>
       <div
         {...otherProps}
-        className={className}
+        className={wrappperClass}
         onClick={preview && !isError ? onPreview : onClick}
-        style={{
-          ...style,
-          width,
-          height,
-        }}
+        style={wrapperStyle}
```

Comment:
height and width should also set in the wrapper element

Reply:
It's broken

![Kapture 2020-10-16 at 16 54 12](https://user-images.githubusercontent.com/14831261/96237790-4d559280-0fd0-11eb-970c-9ac1808accff.gif)